cilium/lb/main.go

```diff
@@ -197,7 +204,12 @@ func cliDumpServices(ctx *cli.Context) {
 	for _, v := range dump {
 		besWithID := []string{}
 		for i, be := range v.BES {
-			str := fmt.Sprintf("%d => %s (%d)", i+1, be.String(), v.FE.ID)
+			var str string
+			if be.Weight != 0 {
```

Comment:
For master service yes be.Weight tells whether that service is weight based or hash based.